### PR TITLE
Clean up push controller js even more

### PR DESCRIPTION
**Pull request by melroy89**
I forgot to remove some additional console.log's. I also added a missing catch.

Also there seems to be a then, without doing anything with the data. Meaning this then might be not needed / can be removed. Can you confirm this?

### CODE REVIEW

File: assets/controllers/push_controller.js [hunk 1]
```diff
@@ -106,7 +103,7 @@ export default class extends Controller {
                 return response.json()
             })
             .then(data => {
-
+                // Nothing?
```
Comment (BentiGorlich) [inline assets/controllers/push_controller.js:106]: I think you can just leave this blank :D

**Review by BentiGorlich — COMMENTED** [review]
apart from this one thing LGTM

File: assets/controllers/push_controller.js (hunk 1 above)
Comment (melroy89) [reply assets/controllers/push_controller.js:106]: If you re not gonna use it, we can just remove this whole then.

Comment (melroy89) [reply assets/controllers/push_controller.js]: Removed the then. It can always be added, if we really want or need to.

**Review by BentiGorlich — APPROVED** [review]

**Pull request merged**